cloud/aws/benchmarks/benchmarks.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# The caller should download the static-tests and extract them into the
+# local directory.
+
+set -x
+
+TEST_FLAGS="-test.bench=.* -test.benchmem -test.run=NONE -test.count=10"
```

Comment:
we only have a handful of tests that are more than 10% slower with benchmem, and some of those have a massive variance in their own runs. I'm not sure that justifies running everything twice. In the long-run, we'll mostly be interested in relative changes to ensure we don't introduce performance degradation.